compass/job/template.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#SBATCH  --job-name={{ job_name }}
+{% if account != '' -%}
+#SBATCH  --account={{ account}}
+{%- endif %}
+#SBATCH  --nodes={{ nodes }}
+#SBATCH  --output={{ job_name }}.o%j
+#SBATCH  --exclusive
+#SBATCH  --time={{ wall_time }}
+{% if qos != '' -%}
+#SBATCH  --qos={{ qos }}
+{%- endif %}
+{% if partition != '' -%}
+#SBATCH  --partition={{ partition }}
+{%- endif %}
+{% if constraint != '' -%}
+#SBATCH  --constraint={{ constraint }}
+{%- endif %}
+
+source load_compass_env.sh
+compass run
```

Comment:
We could add the name of the pickle file to run here.  If you have multiple suites set up in one directory, then as written this will fail.  That said, it's reasonable enough to consider this submission script as a template and expect that a user can/will adjust it for the specific suite/pickle they want to run.  If we do insert the pickle name here, then we probably want to put that in the script name as well so that setting up multiple suites doesn't clobber existing scripts and so it's clear that you are submitting a script specific to a particular suite/pickle.

Reply:
Oh, I can't believe I didn't think of that. I have added it now.  It's pretty obvious that we want that feature.  It isn't included for test case job scripts but it part of suite ones.